setup.py

```diff
@@ -20,6 +20,7 @@
         'cryptography>=1.7.2',
         'pyOpenSSL>=16.2.0',
         'msgpack-python>=0.4.2',
+        'python-magic'
```

Comment:
we may want to very seriously consider supporting the idea of a parser instance on a windows box too...  there may be parser libraries or OS provided parsing capabilities that we want to leverage rather than port to posix.